pytest.ini

```diff
@@ -0,0 +1,39 @@
+[pytest]
+markers =
+    pep8: workaround for https://bitbucket.org/pytest-dev/pytest-pep8/issues/23/
+
+python_files =
+    test_*.py
+
+addopts =
+    --cov ofxstatement.configuration
+		--cov ofxstatement.exceptions
+		--cov ofxstatement.ofx
+		--cov ofxstatement.parser
+    --cov ofxstatement.plugin
+    --cov ofxstatement.statement
+    --cov ofxstatement.tool
+    --cov ofxstatement.ui
+    --cov-report term-missing
+    --no-cov-on-fail
+#    --doctest-modules
+#    --pep8
+#    --flakes
+    --ignore setup.py
+
+pep8ignore =
+    *.py W391
+		test*.py E501
+
+norecursedirs =
+    .svn
+    .git
+    build
+    dist
+    .tox
+    .eggs
```

Comment:
Can we instruct pytest to only look for tests in `src/ofxstatement` so that we don't have to maintain another copy of .gitignore in this obscure place?

Reply:
Well, I got also a coverage on my own plugins here on the system, so I just made all sources explicit.
I have removed the explicit thing and the .gitignore stuff (a copy from somewhere else).